src/fragments/lib/storage/ios/remove.mdx

```diff
@@ -1,5 +1,37 @@
 Delete an object uploaded to S3 by using `Amplify.Storage.remove` and specify the key:
 
```

Comment:
should stop using key terminology here.

Reply:
Suggestion to do:

Delete an object uploaded to S3 by using `Amplify.Storage.remove`. Below then shows the "with" options.